worker/healthcheck/garden.go

```diff
@@ -0,0 +1,69 @@
+package healthcheck
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/pkg/errors"
+)
+
+const containerPayloadFormat = `{"handle":"%s", "rootfs":"raw:///tmp/"}`
```

Comment:
Yeah, I agree it seems odd 😬 It does work, but that's a valid concern.

It creates few `/etc/*` files there (https://github.com/cloudfoundry/guardian/blob/master/rundmc/containerizer.go#L117-L151), although the rundmc implementation does it with an `openat` with `O_CREAT`, but I guess we should not rely on that specific implementation.

Maybe we could go with first creating an empty directory, then passing it there?

Thx!

Reply:
Coupling might make sense in this case though, since we do rely on the behavior you mention around volume readability by unprivileged containers (not sure how they'll fare with `raw:///tmp/` 🤷‍♂️ ). I could see the argument for using BaggageClaim for the directory, regardless of the implementation being a little less simple.